recipe/meta.yaml

```diff
@@ -75,12 +56,13 @@ test:
   imports:
     - sklearn
   commands:
-    - set OPENBLAS_NUM_THREADS=1          # [win]
-    - set OMP_NUM_THREADS=1               # [win]
-    - export OPENBLAS_NUM_THREADS=1       # [not win]
-    - export OMP_NUM_THREADS=1            # [not win]
-    # Skipping testing in Travis-CI for ppc64le pypy
-    - pytest --timeout 300 -n {{ test_cpus }} --verbose --pyargs sklearn -k "not ({{ tests_to_skip }})"    # [not (CI == "travis" and ppc64le and python_impl=="pypy")]
+    {% set tests_to_skip = "_not_a_real_test" %}
+    # https://github.com/scikit-learn/scikit-learn/issues/20335
+    {% set tests_to_skip = tests_to_skip + " or test_loadings_converges" %}
+    # Some tests take alot of memory, and seem to cause segfaults when memory runs out
+    {% set cpus = 8 if aarch64 else (2 if ppc64le else CPU_COUNT) %}
+    - export OPENBLAS_NUM_THREADS=1       # [aarch64]
```

Comment:
I would say those are likely still necessary for x86_64 as well, as otherwise the CPU detection with OpenMP will likely report all physical CPU on the CI runner instead of 2 or so available, leading to CPU over subscription and slower tests..

Reply:
> I would say those are likely still necessary for x86_64 as well

CI is green without them. 🤷 🙃